packages/spatial/src/renderer/components/PostProcessingComponent.tsx

```diff
@@ -59,16 +104,532 @@ export const PostProcessingComponent = defineComponent({
   reactor: () => {
     const entity = useEntityContext()
     const rendererEntity = useScene(entity)
-    const postprocessingComponent = useComponent(entity, PostProcessingComponent)
 
-    useEffect(() => {
-      if (!rendererEntity) return
-      configureEffectComposer(
-        rendererEntity,
-        postprocessingComponent.enabled.value ? postprocessingComponent.effects.get(NO_PROXY_STEALTH) : undefined
-      )
-    }, [rendererEntity, postprocessingComponent.enabled, postprocessingComponent.effects])
+    if (!rendererEntity) return null
 
-    return null
+    return <RendererReactor entity={entity} rendererEntity={rendererEntity} />
   }
 })
+
+const RendererReactor = (props: { entity: Entity; rendererEntity: Entity }) => {
+  const { entity, rendererEntity } = props
+
+  const postprocessingComponent = useComponent(entity, PostProcessingComponent)
+  const camera = useComponent(rendererEntity, CameraComponent)
+  const renderer = useComponent(rendererEntity, RendererComponent)
+  const composer = new EffectComposer(renderer.value.renderer)
```

Comment:
this is creating a new composer and render pass every time the renderer reactor re-runs.

we should use a factory function for the initial state of this reactor like we do starting line 152 with the new scene